docs/frontend-system/utility-apis/04-configuring.md

```diff
@@ -8,74 +8,67 @@ description: Configuring, extending, and overriding utility APIs
 
 Utility APIs are extensions and can therefore optionally be amended with configurability, as well as inputs that other extensions attach themselves to. This section describes how to add those capabilities to your own utility APIs, and how to make use of them as a consumer of such utility APIs.
 
-## Adding configurability
+## Configuring
```

Comment:
the diff here looks bad, but it's mainly because the Adding configurability section was moved to the Creating file ... sorry about that